<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stationPageClass</class>
 <widget class="QMainWindow" name="stationPageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>站点</string>
  </property>
  <property name="windowIcon">
   <iconset resource="stationPage.qrc">
    <normaloff>:/stationPage/src/icon/station.png</normaloff>:/stationPage/src/icon/station.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="myQGraphicsView" name="graphicsView">
      <property name="minimumSize">
       <size>
        <width>275</width>
        <height>400</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>275</width>
        <height>400</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabAdd">
       <attribute name="title">
        <string>添加站点</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>注意</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QTextBrowser" name="textBrowserAdd"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QWidget" name="widget" native="true">
             <layout class="QGridLayout" name="gridLayout">
              <item row="2" column="1">
               <widget class="QComboBox" name="comboBoxBefore"/>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="comboBoxRoute">
                <property name="placeholderText">
                 <string>请选择路线</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="labelBefore">
                <property name="text">
                 <string>前继站点</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="labelAfter">
                <property name="text">
                 <string>后驱站点</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="labelRoute">
                <property name="text">
                 <string>选择路线</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="comboBoxAfter"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_2" native="true">
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="labelName">
                <property name="text">
                 <string>站点名</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEditName"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="labelLongi">
                <property name="text">
                 <string>站点经度</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEditLongi"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="labelLati">
                <property name="text">
                 <string>站点纬度</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="lineEditLati"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonAdd">
             <property name="text">
              <string>确认添加</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabDelete">
       <attribute name="title">
        <string>删除站点</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>注意</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QTextBrowser" name="textBrowserDelete"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_1" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QWidget" name="widget_4" native="true">
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="labelRoute_delete">
                <property name="text">
                 <string>选择路线</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="comboBoxRoute_delete">
                <property name="placeholderText">
                 <string>请先选择路线</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="labelStation_delete">
                <property name="text">
                 <string>选择站点</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="comboBoxStation_delete"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonDelete">
             <property name="text">
              <string>确认删除</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myQGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>myqgraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="stationPage.qrc"/>
 </resources>
 <connections/>
</ui>
